MANIFEST.in

```diff
@@ -37,7 +37,7 @@ include CONTRIBUTING.md
 include *.py
 recursive-include contributions *
 recursive-include bin *
-recursive-include doc *.py *.rst Makefile
+recursive-include doc *.py *.rst Makefile *.txt
```

Comment:
TBH, I'm not quite sure what MANIFEST.in is used for (perhaps when packaging up a release?). Anyway, I've realised that there are rather a lot of (generated) `.txt` files under doc but since it is only the various `pip_requirements.txt` files that I added this for, I've changed it to specify that name.